nixos/modules/tasks/filesystems/vfat.nix

```diff
@@ -11,7 +11,7 @@ in
 {
   config = mkIf (any (fs: fs == "vfat") config.boot.supportedFilesystems) {
 
-    system.fsPackages = [ pkgs.dosfstools ];
+    system.fsPackages = [ pkgs.dosfstools pkgs.mtools ];
```

Comment:
mtools seems pretty obscure and unnecessary for supporting access to FAT filesystems. Why is this needed?

Reply:
The `mtools` utility is used when working with the FAT file system.
See - https://github.com/NixOS/nixpkgs/pull/146288